lib/grape/middleware/versioner/header.rb

```diff
@@ -159,7 +159,7 @@ def version_options
         # header, one can set the `:cascade` option to `false`.
         def cascade?
           if version_options && version_options.key?(:cascade)
-            !!version_options[:cascade]
+            version_options[:cascade]
```

Comment:
Wanted to make sure on these. `:cascade` is going to be set as either `TrueClass` or `FalseClass` as I understood it, which makes the double bang not needed